trojsten/rules/susi.py

```diff
@@ -77,10 +77,11 @@ def prepare_coefficients(self, round):
                 - constants.SUSI_YEARS_OF_CAMPS_HISTORY,
                 going=True,
             )
-            .exclude(Q(event__type__name=constants.SUSI_CAMP_TYPE))
-            .exclude(Q(event__type__name=KMS_MO_FINALS_TYPE))
+            .exclude(event__type__name=constants.SUSI_CAMP_TYPE)
+            .exclude(event__type__name=KMS_MO_FINALS_TYPE)
+            .exclude(type=EventParticipant.ORGANIZER)
```

Comment:
Ked uz som opravoval veci tak som pridal aj sem aby organizatorom nepridavalo koefficient organizovanie inych sustredeni, tak ako je to aj pri susi sustredeniach.